js/settings.js

```diff
@@ -91,6 +91,15 @@ module.exports = function(opts) {
     if (!arguments.length) return settings;
 
     settings = data.grouped.settings;
+    var firstBgTarget = settings[settings.length - 1].bgTarget[0];
```

Comment:
do you think a comment is needed here. Only question I have when I look at the change in context of the rest of the settings is 'why this specific one'. Might just be because this is the first time I have looked at this file